### PR TITLE
Design import export

### PR DESCRIPTION
**Pull request by golbian**
Fixes Issues #31 and #32.

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -1917,6 +1917,23 @@ img.img-cirlce {
     height: 27px;
 }
 
+#dropdownExport1,
+#dropdownExport2,
+#dropdownExport3 {
```
Comment (jajm) [inline public/css/main.css]: It would be better to use a class here, instead of a list of IDs

File: public/css/main.css [hunk 2]
```diff
@@ -1917,6 +1917,23 @@ img.img-cirlce {
...
+    background-color: #eee;
+    list-style: none;
+}
+
+#details1,
+#details2,
+#details3 {
```
Comment (jajm) [inline public/css/main.css]: Same here

File: public/partials/home/sidebar.html [hunk 3]
```diff
@@ -33,6 +33,12 @@
     <a id="usersMainMenu" ng-show="isWSTADMIN" href="/#/users" class="list-group-item">
         <i class="fa fa-user"></i> <span translate>&nbsp;Users  </span><span class="badge">{{counts.users}}</span>
     </a>
+    <a id="usersMainMenu" ng-show="isWSTADMIN" href="/#/import" class="list-group-item">
+        <i class="fa fa-download"></i> <span translate>&nbsp;Import  </span><span class="badge">{{counts.users}}</span>
+    </a>
+    <a id="usersMainMenu" ng-show="isWSTADMIN" href="/#/export" class="list-group-item">
+        <i class="fa fa-upload"></i> <span translate>&nbsp;Export  </span><span class="badge">{{counts.users}}</span>
+    </a>
```
Comment (jajm) [inline public/partials/home/sidebar.html]: Too much copy/paste :-)
The badge makes no sense here

Also, please move `&nbsp;` out of the translatable text

And you're using 'fa-download' for import and 'fa-upload' for export. Shouldn't it be the opposite ?

File: public/partials/io/export.html [hunk 4]
```diff
@@ -1,50 +1,58 @@
...
     <input type="text" ng-model="exportName">
     .json
     <br>
     <button class="btn btn-success" ng-click="downloadExport()">export</button>
     <br>
     <a class="btn btn-default" href="/#/home">back</a>
-    
+  </div>
 </div>
-
```
Comment (jajm) [inline public/partials/io/export.html]: Some tags are not properly indented. Please fix

File: public/partials/io/import.html [hunk 5]
```diff
@@ -119,5 +126,6 @@ <h4>An error has occurred</h4>
         <button ng-click="initImport()">Import new file</button>
         <a class="button" href="/#/home">Back to main page</a>
     </span>
-</div>
 
+</div>
+</div>
```
Comment (jajm) [inline public/partials/io/import.html]: Please fix indentation

**Comment by PaulPoulain** [comment]
testing import: the select file button does not look like a button:
image

**Comment by jajm** [comment]
That's what I see after selecting a JSON file:

Screenshot_2019-05-20 Urungi

@PaulPoulain do you see something in the browser console when you select the .json file ?

**Comment by PaulPoulain** [comment]
yes, after selecting a 'catalogue.json' file, I see:

invalid file format controller.js:52:17
Error: "missing fields"
    onload http://biblibre-urungi.test.biblibre.eu/js/io/controller.js:49

=> this message should be on the screen, it's probably an old .json file (will attach it on this PR)

Trying with a newer json file, it work, even if I get:
Error: Argument 1 of FileReader.readAsText is not an object.
$scope.upload@http://biblibre-urungi.test.biblibre.eu/js/io/controller.js:43:9
fn@http://biblibre-urungi.test.biblibre.eu/js/bundle.min.js line 245 > Function:4:281
m@http://biblibre-urungi.test.biblibre.eu/js/bundle.min.js:8100:8643
i.updateModel/<@http://biblibre-urungi.test.biblibre.eu/js/bundle.min.js:8100:10186
e/<@http://biblibre-urungi.test.biblibre.eu/js/bundle.min.js:141:434
$eval@http://biblibre-urungi.test.biblibre.eu/js/bundle.min.js:156:65
$digest@http://biblibre-urungi.test.biblibre.eu/js/bundle.min.js:153:123
$apply@http://biblibre-urungi.test.biblibre.eu/js/bundle.min.js:156:361
e/q<@http://biblibre-urungi.test.biblibre.eu/js/bundle.min.js:169:386
f@http://biblibre-urungi.test.biblibre.eu/js/bundle.min.js:54:490
mg/k.defer/c<@http://biblibre-urungi.test.biblibre.eu/js/bundle.min.js:57:418

**Comment by PaulPoulain** [comment]
[catalogue.zip](https://github.com/biblibre/urungi/files/3207866/catalogue.zip)

**Comment by jajm** [comment]
> yes, after selecting a 'catalogue.json' file, I see:
> 
> invalid file format controller.js:52:17
> Error: "missing fields"
> onload http://biblibre-urungi.test.biblibre.eu/js/io/controller.js:49
> 
> [...]

Turned this comment into a new issue (#66)

**Pull request closed without merging**